### PR TITLE
Fixed Lunar Battle quest scrolls showing wrong unlock conditions (Issue #10294 )

### LINKED ISSUE #10295: Lunar Battle quest scrolls show wrong unlock conditions
The three Lunar Battle quest scrolls are unlocked after a certain number of check-ins. However for the second and third scrolls, the Quest Shop says that you can unlock them by completing the preceding quests, which is wrong, as well as saying they unlock after check-ins, which is correct. 

In the two screenshots below, only the first sentence is correct. The second sentence should not appear.

image

image

For comparison, the text for the first Lunar Battle quest scroll is correct:
image

### PR DESCRIPTION
**Pull request by DoguD**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #10295

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Even though the Lunar Battle quest scrolls unlocked according to check-in count, in Shop/Quests, locked Lunar Battle Part 2 & 3 shows a message which implies the scroll will be unlocked when the user finishes the previous quest. This message is removed.
Before: 

After:
screen shot 2018-05-03 at 16 51 25

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID: 1b2e5d14-e918-44aa-b669-7c309781fdba

### CODE REVIEW

File: website/client/components/shops/quests/index.vue [hunk 1]
```diff
@@ -139,7 +139,7 @@
                     h4.popover-content-title(v-else) {{ item.text }}
                     .popover-content-text(v-if='item.locked && item.key === "lostMasterclasser1"') {{ `${$t('questUnlockLostMasterclasser')}` }}
                     .popover-content-text(v-if='item.locked && item.unlockCondition && item.unlockCondition.incentiveThreshold') {{ `${$t('loginIncentiveQuest', {count: item.unlockCondition.incentiveThreshold})}` }}
-                    .popover-content-text(v-if='item.locked && item.previous') {{ `${$t('unlockByQuesting', {title: item.previous})}` }}
+                    .popover-content-text(v-if='item.locked && item.previous && item.key !== "moon2" && item.key !=="moon3"') {{ `${$t('unlockByQuesting', {title: item.previous})}` }}
```
Comment (negue) [inline website/client/components/shops/quests/index.vue]: could you move the condition to a method inside of this file, that way if any other quests have to handled like that, the checks are not hidden somewhere in the templates

**Comment by DoguD** [comment]
I have added a new method for doing the check of whether the quest's buying condition is dependent on the completition of a previous quest.
By the way, I am seeing a lot of commits here, did I do something wrong by calling `git fetch upstream`?

**Comment by negue** [comment]
@DoguD this happens if you use `develop` as your work-branch but then with `git fetch upstream` you'll have all commits in that branch ^^

could you try to create a branch and push your work to this and then recreate the pullrequest , its the easier way without rebasing (since a lot more could go wrong there)

**Comment by DoguD** [comment]
https://github.com/HabitRPG/habitica/pull/10346 is the new pull request.

**Pull request closed without merging**